### PR TITLE
Update Readme.md - Reorganize Supported Games

### PR DESCRIPTION
**Pull request by Spiker985**
Replaced separate Protonfixes and Media Foundation lists with a single supported games table.
Added SteamDB and ProtonDB links.
Did not link Arkania as I was unsure which game to select.

Table formatted using Advanced Tables plugin from Obsidian.md.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -61,188 +60,174 @@ Things it contains that Valve's proton does not:
...
+| Far Cry 5                                           | [SteamDB](https://steamdb.info/app/552520)   | [ProtonDB](https://protondb.com/app/552520)  | :heavy_check_mark: | :x:                        |
+| FINAL FANTASY X/X-2 HD Remaster                     | [SteamDB](https://steamdb.info/app/359870)   | [ProtonDB](https://protondb.com/app/359870)  | :heavy_check_mark: | :x:                        |
+| FINAL FANTASY IX                                    | [SteamDB](https://steamdb.info/app/377840/)) | [ProtonDB](https://protondb.com/app/377840)  | :heavy_check_mark: | :x:                        |
+| FINAL FANTASY XIII                                  | [SteamDB](https://steamdb.info/app/292120/)) | [ProtonDB](https://protondb.com/app/292120)  | :heavy_check_mark: | :x:                        |
+| FINAL FANTASY XIV Online                            | [SteamDB](https://steamdb.info/app/39210/))  | [ProtonDB](https://protondb.com/app/39210)   | :heavy_check_mark: | :x:                        |
```
Comment (ianwijma) [inline README.md:134]: This line and the 2 above it have duplicate )) at the end of the steamdb link.

Comment (Spiker985) [reply README.md]: Removed extra )

**Comment by GloriousEggroll** [comment]
Thanks for this but there are two problems: 

(1) I do not want to merge both lists
(2) I do not want to refer to these lists as "supported games" because we are not providing support for them. They are various game fixes which may work at some point, but may also break. I really need to stress here that we are -not- providing support for any games listed, we are just stating that they worked when we tested them with the fixes applied.

Other than those two points reorganizing the list to a cleaner format would be nice and is appreciated.

**Comment by Spiker985** [comment]
I figured I'd put in the PR and you could either keep it or reject it.

I know that I personally have had some trouble in the past when searching
to see if a game was better supported on GE.

I originally intended to leave the two lists separate, but since there were
games that were in both lists - it seemed more appropriate to me to combine
them with columns for the features that have previously had fixes in GE.

If you want, I can split them back apart and replace the original
headings/links. Let me know if you want me to keep the SteamDB and ProtonDB
links too, or if you would rather those go too, and only maintain the
alphabetical order.

After all, you and the other primary contributors are more likely to
maintain the repo and associated games list, not me - so whichever
direction you go is fine with me.

**Comment by Chr1s70ph** [comment]
I really like the newly organized list. As @GloriousEggroll mentioned: "Supported Games" could be misleading. I suggest changing the Section to "Tested Games" with a notice at the top/bottom, that although tested, no guarantee is given for the games to run.

File: README.md [hunk 2]
```diff
@@ -61,188 +60,174 @@ Things it contains that Valve's proton does not:
...
-- Aven Colony
-- American Fugitive
-- Asrtonner
-- Soul Caliber VI
-- Monster Hunter Rise
-- Seven: Days Gone
+- Various upstream WINE patches backported
+- Various wine-staging patches applied as they become needed
+
+## Supported Games
```
Comment (Chr1s70ph) [inline README.md:72]:
```suggestion
## Tested Games
```

File: README.md [hunk 3]
```diff
@@ -61,188 +60,174 @@ Things it contains that Valve's proton does not:
...
+| WORLD OF HORROR                                     | [SteamDB](https://steamdb.info/app/913740)   | [ProtonDB](https://protondb.com/app/913740)  | :x:                | :heavy_check_mark:         |
+| Yakuza 0                                            | [SteamDB](https://steamdb.info/app/638970)   | [ProtonDB](https://protondb.com/app/638970)  | :x:                | :heavy_check_mark:         |
+| Yakuza Kiwami                                       | [SteamDB](https://steamdb.info/app/834530)   | [ProtonDB](https://protondb.com/app/834530)  | :x:                | :heavy_check_mark:         |
+| Yesterday Origins                                   | [SteamDB](https://steamdb.info/app/465280)   | [ProtonDB](https://protondb.com/app/465280)  | :x:                | :heavy_check_mark:         |
+| You Need A Budget 4 (YNAB)                          | [SteamDB](https://steamdb.info/app/227320)   | [ProtonDB](https://protondb.com/app/227320)  | :x:                | :heavy_check_mark:         |
 
 ## Notes
```
Comment (Chr1s70ph) [inline README.md:233]:
```suggestion

- We are *not* providing support for any games listed above. We are just stating that they worked when we tested them with the fixes applied. They may break in upcoming releases.
```

**Comment by Spiker985** [comment]
What about if we cut down the middle of the "Tested versus Supported"
statement and rename it to "Previously Tested Games", and keep the
statement about how the games were tested with previous patches and may
break at any time?

There's no in-house pipeline to confirming all games still function after
updates, right? Because, much like supported games, tested games implies
direct confirmation that they work - at least in my opinion.

I imagine that most of the "patches aren't working" notices come from users
when the game stops working as intended, with the patches applied.

@GloriousEggroll May I request additional input for the above, and in regards to my previous comment as well?

**Pull request merged**